docs/documenter.php

```diff
@@ -122,7 +111,7 @@ function get_namespace_documentation(string $namespace): string
                 }
 
                 $definition_line = get_symbol_definition_line($symbol, $type);
```

Comment:
The definition line for constants always return 0
Maybe we could use better-reflection to guess constants lines in the future.

https://github.com/Roave/BetterReflection/blob/478ed89e695cbff3ac037ccaa713cb56aee359b4/src/Reflector/ConstantReflector.php#L54

Reply:
maybe later, for now it's not really important.